docs/7-Recipes.md

```diff
@@ -0,0 +1,121 @@
+
+### Using standard `logging`
+
+If you prefer to use the standard `logging` library, then you need to do two things.
+By default, logging is disabled, hence you need to enable it to get logging output. In
+addition to that, you need to propagate Loguru messages to standard logging.
+`application_settings` provides a convenience function for this purpose.
+
+```python
+from application_settings import use_standard_logging
+from loguru import logger
+
+use_standard_logging()
+logger.enable("application_settings")
+
+```
+
+The logical location for this statement is before the first invocation of `load` (or, if
+you do not `load` explicitly, before the first invocation of `get`).
+
+## Having a test configuration during testing
+
+During unit testing, you typically would like to have a specific test configuration and
+it would be a hassle to provide a file for this purpose. this especially holds true
+for the situation in which you have used `application_settings` to define a config
+section for your library package, and hence have no config container in your package
+that can do the file handling.
+
+Fortunately, there is another way, namely by using classmethod `set`. As an example,
+let's create a test config for `ExampleConfigSection`, (see
+[Quick Start](Quick_start.md)):
+
+```python
+# Create a (nested) dictionary that holds the test values for the relevant fields
+test_config = {
+    "field1": 3.3,
+    "field2": -2
+}
+
+# Invoke classmethod set on the Config(Section) with this dict as parameter;
+# this will set the config singleton with the provided values
+ExampleConfigSection.set(test_config)
+```
+
+The `test_config` dictionary only has to contain values for fields that have no default
+and for fields for which the test value differs from the default one.
+
+## Initialization needs to depend on configuration
+
+### Application case
+
+The situation may occur that your application imports a module that holds code that is
+initialized during import and you want this initialization to be configurable. For
+example, you might want a configurable initial value for a module global variable
+or a class variable.
+
+To make this work, you need to assure that the configuration is loaded before the
+module of concern is imported. To make this robust, the following way of work is
+suggested:
+
+- Define your configuration class(es) in a separate module (or multiple separate
+  modules);
+- In the module that defines your configuration container, set `config_filepath_from_cli`
+  and `load` the configuration;
+- Make sure that the module with your config container is `import`ed before any of the
+  configured global module variables and/or class variables are initialized. One way of
+  assuring this in a robust way is by `import`ing your config container at the top of the
+  file that is the entry point of your application (i.e., typically `__main__.py`);
+- Start your application with the filepath to the config file as command line parameter.
```

Comment:
It's for me a bit hard to exactly vision what you're trying to explain here. Does it make sense to add an example in the `examples` directory. Just some minimal example that shows what you're trying to do?

Reply:
Example added